### PR TITLE
Ew Add index to device type (CU-86dthbn9e)

### PR DESCRIPTION
**Pull request by elanawood**
Added an index to the device type column within the devices table.

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86dthbn9e Add index to sensor db for device_type column](https://app.clickup.com/t/86dthbn9e)

**Comment by elanawood** [comment]
Getting sudo apt get update errors on Travis :(

File: server/db/048-add-index-to-deviceType-column [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    migrationId := 48;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        
+        CREATE INDEX deviceType_index ON devices(device_type);
```
Comment (micahdbak) [inline server/db/048-add-index-to-deviceType-column:15]: Ah, this file's name should end with ".sql".

**Review by micahdbak — COMMENTED** [review]
One change then I think this PR is ready to merge!

File: server/db/048-add-index-to-deviceType-column (hunk 1 above)
Comment (elanawood) [reply server/db/048-add-index-to-deviceType-column:15]: Fixed!

**Comment by micahdbak** [comment]
@elanawood , please merge main into this branch one more time to check that all test cases pass.

File: server/db/048-add-index-to-deviceType-column [hunk 2]
```diff
@@ -0,0 +1,20 @@
+DO $migration$
```
Comment (micahdbak) [inline server/db/048-add-index-to-deviceType-column:1]: This file should be deleted; the .sql file can be kept

**Review by micahdbak — COMMENTED** [review]
SQL file looks good!

**Review by micahdbak — APPROVED** [review]
Looks good!

**Pull request merged**